stix_shifter_modules/crowdstrike/stix_transmission/api_client.py

```diff
@@ -43,6 +43,7 @@ def get_detections_IDs(self, filter, limit, sort=None):
         data = dict()
         headers['Content-Type'] = 'application/json'
         headers['Accept'] = 'application/json'
+        headers['user-agent'] = 'ibm_stixshifter_1.0'
```

Comment:
I think the user-agent needs to be more generic not ibm specific since connector can be used by non ibm products. let us know if you have any specific reason to set that name.

Reply:
How about oca_stixshifter_1.0 ?

Reply:
does 1.0 mean anything? if not then i would remove that.

Reply:
Yes, it is part of our API standards at CRWD.  As we contribute to the CrowdStrike module we will update it.  It helps in the long term when we are troubleshooting issues that way we know which clone they are working from.

Reply:
Sounds good.